app/components/PageTitle.js

```diff
@@ -1,15 +1,21 @@
 // @flow
 import * as React from "react";
+import { observer, inject } from "mobx-react";
 import { Helmet } from "react-helmet";
+import AuthStore from "stores/AuthStore";
 
 type Props = {
   title: string,
   favicon?: string,
+  auth: AuthStore,
 };
 
-const PageTitle = ({ title, favicon }: Props) => (
+const { auth } = this.props;
+const { team } = auth;
+
+const PageTitle = observer(({ auth, title, favicon }: Props) => (
   <Helmet>
-    <title>{`${title} - Outline`}</title>
+    <title>{`${title} - ${team.name} | Outline` | `${title} - Outline`}</title>
```

Comment:
Before the `|` will always be a string, and therefore always be truthy, the second part of the statement will never be executed